server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -1160,7 +1167,8 @@ protected Node(
                 resourceUsageCollectorService,
                 segmentReplicationStatsTracker,
                 repositoryService,
-                admissionControlService
+                admissionControlService,
+                peerRecoveryTargetService
```

Comment:
Is this the only way to supply the peerRecoveryTargetService to the NodeService class?